### PR TITLE
feat: winner design update

### PR DESCRIPTION
**Pull request by Suraj975**
### Changes

- [ ]Winner design update
- [ ]
- [ ]

### Affected Platforms

- [ ] Android
- [ ] iOS
- [ ] Web

### Tested on Platforms

- [ ] Android
- [ ] iOS
- [ ] Web

### Checks

- [ ] Build

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -6,11 +6,13 @@ import Home from './components/Home';
 import GameRoom from './components/GameRoom';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import JoinGame from './components/JoinGame';
+// import Winner from './components/Win'
```
Comment (sinharaksh1t) [inline src/App.tsx:9]: remove, please?

File: src/App.tsx [hunk 2]
```diff
@@ -6,11 +6,13 @@ import Home from './components/Home';
...
 
 const App: React.FC = () => {
   return (
     <Router>
       <div className="App">
+        {/* <Winner /> */}
```
Comment (sinharaksh1t) [inline src/App.tsx:15]: remove, please?

File: src/components/Win.tsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
+import React, { ReactElement } from 'react';
+import styled from 'styled-components';
+import HALLOWEEN from '../assets/halloween.gif';
+import { Container, Image, GameWrapper, SharingMethods } from './Styles';
+import WINNER from '../assets/Winner.png';
+import { shareButton, colors } from '../constants/index';
+
+const MiddleWrapper = styled.div`
```
Comment (sinharaksh1t) [inline src/components/Win.tsx:8]: Move this to `Styles.tsx,` please?

File: src/components/Win.tsx [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+  color: ${colors.buttonTextColor};
+  img {
+  }
+  .kingsLayer {
+    position: absolute;
+    margin-top: 11 0px;
+    margin-left: -68px;
+  }
+`;
+const Share = styled.p`
```
Comment (sinharaksh1t) [inline src/components/Win.tsx:26]: Move this to `Styles.tsx,` please?

File: src/components/Win.tsx [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+  margin: 8px 20px 0px 0px;
+`;
+
+const TextContent = styled.div`
```
Comment (sinharaksh1t) [inline src/components/Win.tsx:30]: Move this to `Styles.tsx,` please?

**Review by sinharaksh1t — COMMENTED** [review]
If there are UI changes in this, can we have some screenshots, please?

File: src/components/Win.tsx [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+      <SharingMethods
+        data-href="http://localhost:3000/game"
+        data-layout="button_count"
+        data-size="small"
+      >
+        {' '}
+        <Share>Share</Share>
+        {Object.keys(shareButton).map(
+          (name: any, i: number): ReactElement => (
+            <a key={i} target="_blank" rel="noopener noreferrer" href={shareButton[name].href}>
```
Comment (sinharaksh1t) [inline src/components/Win.tsx:83]: what's `shareButton[name].href`

Comment (Suraj975) [reply src/components/Win.tsx:83]: Share button is an object with names of social media platform and href determining redirect link to those platform. I ll change the href to url for better clearity.

Comment (sinharaksh1t) [reply src/components/Win.tsx:83]: roger that! Thanks for the explanation.

File: src/App.tsx (hunk 2 above)
Comment (Suraj975) [reply src/App.tsx]: Currently, we don't know where to place this component, that's the only reason for commenting it out.

File: src/components/Win.tsx (hunk 3 above)
Comment (Suraj975) [reply src/components/Win.tsx]: done

File: src/components/Win.tsx (hunk 4 above)
Comment (Suraj975) [reply src/components/Win.tsx]: done

File: src/components/Win.tsx (hunk 5 above)
Comment (Suraj975) [reply src/components/Win.tsx]: done

**Review by sinharaksh1t — APPROVED** [review]

**Pull request merged**